src/applications/appeals/995/constants.js

```diff
@@ -101,9 +101,6 @@ export const errorMessages = {
     upload: 'You must provide a password to decrypt this file',
   },
 
-  missingEmail: 'You must provide an email address',
-  missingPhone: 'You must provide a home or mobile phone number',
-  missingAddress: 'You must provide an address',
```

Comment:
Were these unused? Or is this now taken care of in a different file?

Reply:
Unused now. The error messages have been moved into [platform profile content](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms-system/src/js/utilities/data/profile.js#L77).